src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs

```diff
@@ -0,0 +1,175 @@
+        public string FullName => $"{FirstName} {LastName}";
+        private string? FirstName { get; init; }
+        public string? MiddleName { get; set; }
+        private string? LastName { get; init; }
+        public string? PhoneNumber { get; init; }
+        private DateTime? DateOfBirth { get; init; }
+        public string BirthDate => DateOfBirth.HasValue ? DateOfBirth.Value.ToString("dd MMM yyyy") : string.Empty;
+        public AddressDetailsSection? Address { get; init; } = new();
+
+        public static implicit operator CandidateDetailsSection(GetApplicationSummaryQueryResult.CandidateDetailsSection source)
+        {
+            return new CandidateDetailsSection
+            {
+                Id = source.Id,
+                GovUkIdentifier = source.GovUkIdentifier,
+                Email = source.Email,
+                FirstName = source.FirstName,
+                MiddleName = source.MiddleName,
+                LastName = source.LastName,
+                PhoneNumber = source.PhoneNumber,
+                DateOfBirth = source.DateOfBirth,
+                Address = source.Address
+            };
+        }
+    }
+
+    public class AddressDetailsSection
+    {
+        public string? AddressLine1 { get; init; }
+        public string? AddressLine2 { get; init; }
+        public string? Town { get; init; }
+        public string? County { get; init; }
+        public string? Postcode { get; init; }
+
+        public static implicit operator AddressDetailsSection?(GetApplicationSummaryQueryResult.AddressDetailsSection? source)
+        {
+            if (source is null) return null;
+
+            return new AddressDetailsSection
+            {
+                AddressLine1 = source.AddressLine1,
+                AddressLine2 = source.AddressLine2,
+                Town = source.Town,
+                County = source.County,
+                Postcode = source.Postcode,
+            };
+        }
+    }
+
+    public bool IsDisabilityConfident { get; init; }
+
+    public bool IsApplicationComplete => EducationHistory is {TrainingCoursesStatus: SectionStatus.Completed, QualificationsStatus: SectionStatus.Completed} &&
+                                         WorkHistory is {VolunteeringAndWorkExperienceStatus: SectionStatus.Completed, JobsStatus: SectionStatus.Completed} &&
+                                         (ApplicationQuestions.AdditionalQuestion1Id is not null || ApplicationQuestions.AdditionalQuestion1Status == SectionStatus.Completed) &&
+                                         (ApplicationQuestions.AdditionalQuestion2Id is not null || ApplicationQuestions.AdditionalQuestion2Status == SectionStatus.Completed) &&
+                                         InterviewAdjustments.RequestAdjustmentsStatus == SectionStatus.Completed &&
+                                         (!IsDisabilityConfident || DisabilityConfidence.InterviewUnderDisabilityConfidentStatus == SectionStatus.Completed);
```

Comment:
One for later but I feel this should come from the candidate API, it knows all the statuses and can also determine if the application can be submitted.